Verifying sv/ldv-consumption/32_7a_cilled_true-unreach-call_linux-3.8-rc1-drivers--misc--pch_phub.ko-main.cil.out.c
main using GenerateFunctionCallPreprocessor ...
GenerateFunctionCallPreprocessor parents for __VERIFIER_error:
__VERIFIER_error
  ldv_error
    ldv_atomic_dec_and_mutex_lock_pch_phub_mutex
    <end>
    ldv_mutex_trylock_pch_phub_mutex
    <end>
    ldv_mutex_lock_pch_phub_mutex
    <end>
    ldv_mutex_lock_killable_pch_phub_mutex
    <end>
    ldv_atomic_dec_and_mutex_lock_mutex_of_device
    <end>
    ldv_mutex_lock_killable_mutex_of_device
    <end>
    ldv_mutex_lock_interruptible_mutex_of_device
    <end>
    ldv_atomic_dec_and_mutex_lock_lock
    <end>
    ldv_mutex_trylock_lock
    <end>
    ldv_mutex_lock_killable_lock
    <end>
    ldv_mutex_lock_interruptible_lock
    <end>
    ldv__builtin_trap
    <end>
    ldv_mutex_unlock_pch_phub_mutex
      ldv_mutex_unlock_11
        pch_phub_bin_write
          main
          <end>
      ldv_mutex_unlock_10
        pch_phub_bin_write
          main
          <end>
      ldv_mutex_unlock_8
        pch_phub_bin_read
          main
          <end>
      ldv_mutex_unlock_7
        pch_phub_bin_read
          main
          <end>
    ldv_mutex_lock_interruptible_pch_phub_mutex
      ldv_mutex_lock_interruptible_9
        pch_phub_bin_write
          main
          <end>
      ldv_mutex_lock_interruptible_6
        pch_phub_bin_read
          main
          <end>
    ldv_mutex_unlock_mutex_of_device
      ldv_mutex_unlock_5
      <end>
    ldv_mutex_trylock_mutex_of_device
      ldv_mutex_trylock_4
      <end>
    ldv_mutex_lock_mutex_of_device
      ldv_mutex_lock_3
      <end>
    ldv_mutex_unlock_lock
      ldv_mutex_unlock_2
      <end>
    ldv_mutex_lock_lock
      ldv_mutex_lock_1
      <end>
    ldv_check_final_state
      main
      <end>
    ldv_check_final_state
      main
      <end>
    ldv_check_final_state
      main
      <end>
